feature/#508 remove pipeline config doc and added sequence doc to scenario config (#538)
Avaiga/taipy-doc: docs/manuals/core/config/advanced-config.md

    software project (from initial development to production).

>     1. To start, as a developer I will design my application by configuring all the new entities  (data

Reviewer: ```suggestion
    1. As a developer, I will design my application by configuring all the new entities (data
```

>     nodes, tasks, scenarios) using the Python code configuration with just a minimal number of attributes

Reviewer: ```suggestion
    nodes, tasks, scenarios) using the Python code configuration with just a few attributes
```

# Python code configuration

The configuration (for all your data nodes, tasks, scenarios, etc.) can be defined directly using Python
code. This configuration can be done using methods from the `Config^` class. This Python configuration is meant to
be used during the application development phase. It overrides the default configuration:

> if some values are not provided, the default configuration applies.

Reviewer: ```suggestion
the default configuration applies if some values are not provided.
```

        outputs = [ "orders:SECTION",]
        skippable = "False:bool"

        [SCENARIO.scenario_configuration]
        tasks = [ "training:SECTION", "predicting:SECTION", "planning:SECTION",]
        frequency = "MONTHLY:FREQUENCY"

        [SCENARIO.scenario_configuration.comparators]
        sales_predictions = [ "functions.compare:function",]

        [SCENARIO.scenario_configuration.sequences]
        sales = [ "training:SECTION", "predicting:SECTION",]
        production = [ "planning:SECTION",]
        ```

>         Note that the type of the non-string configuration attributes is specified in the _TOML_ file by adding at the

Reviewer: ```suggestion
        Note that the type of the non-string configuration attributes is specified in the _TOML_ file by adding them
```

Author: the whole sentence is actually: "Note that the type of the non-string configuration attributes is specified in the _TOML_ file by adding at the end of the value (':bool', ':int', or ':float')."

So I don't think this change will be good 😃
